torch/_dynamo/variables/builder.py

```diff
@@ -1582,11 +1588,17 @@ def wrap_module(self, value: torch.nn.Module):
             self.install_guards(GuardBuilder.TYPE_MATCH)
             if torch._dynamo.config.inline_inbuilt_nn_modules:
                 freezing = is_parameter_freezing()
-                for p in value.parameters():
-                    self.mark_static_input(p, guard=freezing)
-
-                for b in value.buffers():
-                    self.mark_static_input(b, guard=freezing)
+                # Guard against the case where user may overwrite named parameters
+                # / named buffers
+                # NOTE: This is not likely to happen but worth guarding to avoid
+                # exception
+                if value.named_parameters == og_module_named_parameters_fn_ptr:
```

Comment:
Can you put a breakpoint and check that this codepath is taken when there is no overriding? I think the test tests the other way around. I am concerned that you need `__func__` attribute to get the function from the method.

Reply:
Also use `is` instead of `==` here because we are checking that the objects are essentially same.